### PR TITLE
Switch to a bedrock geoip service using our CDN headers

### PR DESCRIPTION
**Pull request by pmac**
https://support.cloudflare.com/hc/en-us/articles/200168236-What-does-CloudFlare-IP-Geolocation-do-

### CODE REVIEW

**Comment by pmac** [comment]
This should work with all of our code that currently works with mozilla location services. It won't work for demos or www-dev or any other instance that isn't behind CloudFlare, so that is definitely a down side. I could make it so that it could support both depending on a setting, but that would be a bit more code (not a lot, but more, and this seemed time sensitive). I can always followup later with that improvement and tests for the view.

**Comment by pmac** [comment]
Hmm... thinking about it, I should add the service via this PR, then change the endpoints in another so that we can test the thing manually before putting it into use. I'll split it out now.

**Comment by pmac** [comment]
Okay, now this PR only adds the new geolocation endpoint (`/country-code.json`). We can get this into stage and prod (the two instances behind CloudFlare) for testing if we do indeed want/need it. After that a PR to change over to using should be easy.

File: bedrock/base/views.py [hunk 1]
```diff
@@ -3,12 +3,30 @@
...
+
+    https://support.cloudflare.com/hc/en-us/articles/200168236-What-does-CloudFlare-IP-Geolocation-do-
+    """
+    country_code = request.META.get('HTTP_CF_IPCOUNTRY', 'XX')
+    if country_code == 'XX' or len(country_code) != 2:
+        return HttpResponseJSON({'country_code': None}, status=404)
+
+    return HttpResponseJSON({
+        'country_code': country_code,
+    })
```
Comment (jpetto) [inline bedrock/base/views.py]: Wondering if we could put an easy short-circuit here for local dev. Maybe have it look for a config setting? Or if `DEBUG=True` just return `us`?

Comment (pmac) [inline bedrock/base/views.py]: Could definitely do that.... let me think on this...

**Comment by pmac** [comment]
@jpetto went ahead and added a couple tests just to be safe :) Thanks for the r?

**Comment by jpetto** [comment]
:shipit:

**Comment by pmclanahan** [comment]
Circle having problems. I'm going to override the check and merge since this doesn't affect any prod pages.

**Pull request merged**